images/monitoring/Dockerfile

```diff
@@ -15,16 +15,16 @@
 FROM golang:1.10.2
 RUN apt-get update
 
-ENV TEMP_TOOLS_REPO_DIR /go/src/github.com/knative/test-infra/tools
+ENV TEMP_REPO_DIR /go/src/github.com/knative/test-infra
```

Comment:
any reason for this?

Reply:
Yes, after the mysql part is moved to <project-root>/shared, it can't find the mysql stuff since it requires files not in the tools directory. 

Note: I should really update the build test so it checks that monitoring image builds correctly.

Reply:
ah. okies.